Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,93 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            var i = Console.ReadLine();
+            var isNumeric = int.TryParse(i, out int number);
+            if (String.IsNullOrEmpty(i))
+            {
+                return 0;
+            }
+            else if (!isNumeric)
+            {
+                Console.Write($"\"{i}\" is not a valid number.");
+                return -1;
+            }
+            else if (isNumeric && number == 0)
+            {
+                Console.Write($"\"{i}\" is not a valid number.");
+                return 0;
+            }
+            return number;
         }
 
         public static string PromptString(string message)
         {
-            // To do: call your implementation. 
-            return "";
-        }
+            Console.WriteLine(message);
+            var s = Console.ReadLine();
```

Comment:
Avoid using variable names with a single character. Calling it "input" is even better than calling it string (you would need to add @ symbol to allow variable names after keywords)